### PR TITLE
Fix modal backdrop overlaying the modal's scrollbar

### PR DESCRIPTION
**Pull request by hnrch02**
Fixes #14916.
Fixes #15008.

/cc @cvrebert

### CODE REVIEW

File: less/modals.less [hunk 1]
```diff
@@ -62,10 +62,9 @@
 
 // Modal background
 .modal-backdrop {
-  position: fixed;
+  position: absolute;
   top: 0;
   right: 0;
-  bottom: 0;
```
Comment (prettycode) [inline less/modals.less]: +1

Removing `bottom: 0;` causes the calculated height of `.modal-backdrop` to be 0.

**Comment by ollietb** [comment]
This change also breaks my website

**Comment by dbrin** [comment]
Modal back drop is deffinitely broken after this CSS tweak:  

q3cug29 1

File: less/modals.less (hunk 1 above)
Comment (dbrin) [inline less/modals.less]: +1 this is now broken

**Comment by hnrch02** [comment]
Guys, this has been fixed by #15881.

**Pull request merged**